<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyCat_GUI</class>
 <widget class="QDialog" name="CopyCat_GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>496</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>496</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CopyCat Setup</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="main_layout">
    <item>
     <layout class="QHBoxLayout" name="gt_frames">
      <item>
       <widget class="QLabel" name="gt_label">
        <property name="text">
         <string>Ground Truth Sequence:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="gt_spacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="gt_file_path">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>path to first frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gt_file_browse">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="input_frames">
      <item>
       <widget class="QLabel" name="input_label">
        <property name="text">
         <string>Input Sequence:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="input_spacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="input_file_path">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>path to first frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="input_file_browse">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="data_file_browse">
      <item>
       <widget class="QLabel" name="data_label">
        <property name="text">
         <string>Data Directory:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="data_spacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="data_dir_path">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>path to model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="data_dir_browse">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="script">
      <item>
       <widget class="QLabel" name="script_label">
        <property name="text">
         <string>Nuke Script Directory:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="script_spacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="script_path">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>path to .nk file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="script_path_browse">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="top_spacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="cc_header">
      <item>
       <spacer name="cc_header_spacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="cc_header_label">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CopyCat Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="cc_header_spacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="middle_spacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="cc_settings">
      <item>
       <layout class="QVBoxLayout" name="left_settings">
        <item>
         <layout class="QHBoxLayout" name="gpu_options">
          <item>
           <spacer name="gpu_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="gpu_switch">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Use GPU:</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="epoch_settings">
          <item>
           <widget class="QLabel" name="epoch_label">
            <property name="text">
             <string>Epochs:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="epoch_spacer1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="epoch_value">
            <property name="text">
             <string>10000</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="size_settings">
          <item>
           <widget class="QLabel" name="model_label">
            <property name="text">
             <string>Model Size: </string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="model_spacer1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="model_size">
            <item>
             <property name="text">
              <string extracomment="Quick Training">Small</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string extracomment="Regular Training">Medium</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string extracomment="Detailed Training">Large</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="crop">
          <item>
           <widget class="QLabel" name="crop_label">
            <property name="text">
             <string>Crop Size:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="crop_spacer1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="crop_size">
            <item>
             <property name="text">
              <string>32</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>64</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>128</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>256</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>512</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="cc_spacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="right_settings">
        <item>
         <layout class="QHBoxLayout" name="batch_type">
          <item>
           <widget class="QLabel" name="batch_type_label">
            <property name="text">
             <string>Batch Size:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="batch_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="batch_switch">
            <item>
             <property name="text">
              <string extracomment="Automatically creates batch sizes based on available VRAM">Auto</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string extracomment="Manual batch sizing">Manual</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="batch_manual">
          <item>
           <widget class="QLabel" name="batch_manual_label">
            <property name="text">
             <string>Manual Size:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="batch_manual_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="batch_manual_size"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="bottom_spacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="progress_checks">
      <item>
       <layout class="QHBoxLayout" name="contact_sheet">
        <item>
         <widget class="QLabel" name="contact_sheet_label">
          <property name="text">
           <string>Contact Sheet:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="contact_sheet_spacer1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="contact_sheet_quantity">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="checkpoint">
        <item>
         <widget class="QLabel" name="checkpoint_label">
          <property name="text">
           <string>Checkpoint:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="checkpoint_spacer1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="checkpoint_quantity">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>1000</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="command_line">
      <item>
       <widget class="QLabel" name="command_line_label">
        <property name="text">
         <string>CLI:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="command_line_command">
        <property name="text">
         <string>Paste the command generated here into your terminal</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="generate_button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>450</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="credit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Davis Hardy 2023</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
